data.sql

```diff
@@ -86,6 +86,138 @@ INSERT INTO
     )
 VALUES
     ('Angemon', '2005-06-12', 1, true, -45.0),
-('Boarmon', '2005-06-07', 7, true, -20.4),
-('Blossom', '1998-10-13', 3, true, -17.0),
-('Ditto', '2022-05-14', 4, true, -22.0);
\ No newline at end of file
+    ('Boarmon', '2005-06-07', 7, true, -20.4),
+    ('Blossom', '1998-10-13', 3, true, -17.0),
+    ('Ditto', '2022-05-14', 4, true, -22.0);
+
+INSERT INTO
+    vets (full_name, age)
+VALUES
+    ('Sam Smith', 34);
+
+INSERT INTO
+    vets (full_name, age)
+VALUES
+    ('Jennifer Orwell', 19);
+
+INSERT INTO
+    vets (full_name, age)
+VALUES
+    ('Bob', 45);
+
+INSERT INTO
+    vets (full_name, age)
+VALUES
+    ('Melody Pond', 77);
+
+INSERT INTO
+    vets (full_name, age)
+VALUES
+    ('Dean Winchester', 14);
+
+INSERT INTO
+    vets (full_name, age)
+VALUES
+    ('Jodie Whittaker', 38);
```

Comment:
- [ ] Nice work writing the SQL code to insert data into a table :clap: However according to the project requirement, these data should be inserted into the `owners` tabe (see image). Therefore, kindly insert these data into the `owners` table instead of the `vets` to follow the project requirement.
![Screenshot from 2023-09-30 01-09-08](https://github.com/ZabihullahNooriWardak/vet-Clinic/assets/91142432/9f5b62b1-dddf-4a3e-92c1-ae5eaf4e48b6)

I suggest you remove these data from the `vets` table and add them to the `owners` table